application/models/Permission.php

```diff
@@ -677,16 +677,26 @@ public function hasSurveyPermission($iSurveyID, $sPermission, $sCRUD = 'read', $
     }
 
     /**
-     * Returns true if a role has permission to read/create/update a certain template
-     * @param string $roleId
-     * @param $sCRUD string The permission detailsyou want to check on: 'create','read','update','delete','import' or 'export'
-     * @param integer $iUserID integer User ID - if not given the one of the current user is used
-     * @return bool True if user has the permission
+     * Returns true if a role has permission for crud
+     * @param integer $roleId
+     * @param string $sPermission 
+     * @param string $sCRUD The permission detailsyou want to check on: 'create','read','update','delete','import' or 'export'
+     * @return bool allwoedpermssion
      */
-    public function hasRolePermission($iRoleId, $sPermission, $sCRUD = 'read')
+    public function RoleHasPermission($iRoleId, $sPermission, $sCRUD = 'read')
```

Comment:
Not PSR-12 compliant name, should start with lower letter.

Reply:
Oups …